modules/govuk/manifests/apps/performanceplatform_big_screen_view.pp

```diff
@@ -29,5 +34,17 @@
       ssl_only        => true,
       app_port        => 3058,
     }
+    # Since this is not a full app, we need to ensure env dir exists.
+    file { "/etc/govuk/${app_name}/env.d":
+      ensure  => 'directory',
+      purge   => true,
+      recurse => true,
+      force   => true,
+    }
+    govuk::app::envvar { "${app_name}-PUBLISHING_API_BEARER_TOKEN":
+      app     => $app_name,
+      varname => 'PUBLISHING_API_BEARER_TOKEN',
+      value   => $publishing_api_bearer_token,
```

Comment:
Final thing (sorry this is such a hassle :disappointed:), this `govuk::app::envvar` needs a parameter of `notify_service => false,` because otherwise the `govuk::app::envvar` defined type will attempt to notify the service `performanceplatform-big-screen-view` which doesn't exist:

```
Error: Could not find dependent Govuk::App::Service[performanceplatform-big-screen-view] for File[/etc/govuk/performanceplatform-big-screen-view/env.d/PUBLISHING_API_BEARER_TOKEN] at /var/govuk/govuk-puppet/modules/govuk/manifests/app/envvar.pp:24
```